pgbot/commands/user.py

```diff
@@ -12,6 +12,7 @@
 
 from pgbot import clock, common, docs, embed_utils, emotion, sandbox, utils
 from pgbot.commands.base import BaseCommand, CodeBlock, HiddenArg, String
+from pgbot.utils import *
```

Comment:
Just get rid of this wildcard import while fixing the merge conflict (That is, reject your change and accept incoming change) ;)